app/src/main/java/org/xwiki/android/sync/activities/AccountListAdapter.kt

```diff
@@ -27,12 +27,8 @@ class AccountListAdapter(
 
     override fun onBindViewHolder(holder: AccountListViewHolder, position: Int) {
         val account = availableAccounts.get(position)
-        holder.tvAccountName.text = account.accountName
-        holder.tvAccountServerAddress.text = account.serverAddress
 
-        holder.llAccountItem.setOnClickListener {
-            listener(account)
-        }
+        holder.currentAccount(account, listener)
     }
 
     class AccountListViewHolder(view: View) : RecyclerView.ViewHolder(view) {
```

Comment:
Variables (except of `account`) should be private inside of `AccountListViewHolder`

Reply:
okay